Core.DomainServices/Queries/SystemUsage/QueryReadModelsWhichShouldBecomeActive.cs

```diff
@@ -29,7 +29,12 @@ public IQueryable<ItSystemUsageOverviewReadModel> Apply(IQueryable<ItSystemUsage
                             // Include only if not expired or no expiration defined
                             (x.SourceEntity.ExpirationDate == null || currentTime <= x.SourceEntity.ExpirationDate)
                         )
-                    )
+                    ) &&
+                    x.MainContractIsActive == false &&
```

Comment:
this is not good enough... take a look at the expression in the similar query for it-contract... it really is what we are after here

- If there is a contract and IF the state of the contract can influence the state of the it-system THEN update it.. the IsActive is just the "enforce flag".